portfolio/src/main/webapp/index.html

```diff
@@ -6,84 +6,73 @@
+          <p>
+            Besides the Unity project that I mentioned above, another project that I did that I find particularly interesting is Zing. 
+            Zing is a project that I started with two people over the summer my freshman year when I did an internship at Life Changing Labs in Ithaca. 
+            Life Changing Labs is an entrepreneurship program which matches founders with interns that they work with to build their startup idea. 
+            Zing is a project that brings isolated students on campus together and during this time of quarantine we actually made a variant of Zing called 
+            Quarantine Buddy which pairs users to do a Zoom call together so they don't have to feel as isolated when they are at home. A special 
+            shoutout to my teammate Sam who got the opportunity to be on ABC news with Kelly and Ryan to talk about Quarantine Buddy.
+            <br/><br/>
+            I also worked on a bomb defusal game with the C programming language on a single FRDM MK64F board with my friend. 
+            We used the board's built-in accelerometer to determine orientation values for displaying the different sections of the game. 
+            Each section had a different mini-game to play: Morse code translation, simon says, reaction time, and a last stand game which depended on the results of the other 3 mini-games. 
+            The game uses the LED to display a randomly selected sequence of colors based on the rules of each mini-game and the user uses the pushbuttons to play the game.
+          </p>
         </div>
-        <div id="image_label">
-            <a href="/images/MangoMangoSweets.jpg">
-                <img src="/images/MangoMangoSweets.jpg" width=250 alt="Mango Mango dessert">
-            </a>
-            <p>Yum! I have a sweet tooth.</p>
+        <div class="section" id="bottom-section">
+          <div id="link-section">
+            <h2>Links</h2>
+            <a href="https://github.com/ALai2"><p>Github</p></a>
+            <a href="https://www.linkedin.com/in/alai2"><p>LinkedIn</p></a>
+          </div>
+          <div>
+            <h2>Fact Hat</h2>
+            <p>RANDOM FACT &#8594; <span id="fact-container">???</span></p>
+            <button id="fun-button" onclick="addRandomFact()">Click Me!</button>
+          </div>
         </div>
-        <div id="image_label">
-            <a href="/images/MCStarterHouse.jpg">
-                <img src="/images/MCStarterHouse.jpg" width=250 alt="My first Minecraft starter house">
-            </a>
-            <p>My first starter house in Minecraft.</p>
-        </div>
-
-      </div>
-
-      <h3>Why These Pictures?</h3>
-      <h2>My Hidden Talent</h2>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+        <button class="page-button" onclick="window.location.href='/images.html'"><strong>Go to Gallery!</strong></button>
```

Comment:
Move 2 spaces to the left.

Reply:
I committed a change which fixed the indentations that I missed before.